libs/components/src/lib/login-form/login-form.tsx

```diff
@@ -0,0 +1,115 @@
+import { useEffect } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Button, Form, Typography } from 'antd';
+import { useLoginUserMutation } from 'src/redux/auth/auth-api';
+import { setUser } from 'src/redux/auth/auth-slice';
+import * as yup from 'yup';
+
+import { StyledInput, StylesButton } from './styles';
+
+import 'antd/dist/antd.css';
```

Comment:
it should be only once in index file of root

Reply:
done

Reply:
I still see it here